pyproject.toml

```
@@ -26,7 +26,7 @@ dependencies = [
     "numcodecs>=0.15.1",
     "ujson",
     "packaging",
-    "zarr>=3.0.2",
+    #"zarr>=3.0.2",
```

```suggestion
    "zarr>=3.0.2",
```

bring back so it uses dependency-groups